index.js

```diff
@@ -116,27 +132,50 @@ btnClear.addEventListener('click', function () {
     nr1 = 0;
     nr2 = 0;
     operation = undefined;
+    calcScreen.innerHTML = 0;
     console.log(nr1);
 
 });
 
 btnSubtraction.addEventListener('click', function () {
     operation = '-';
+    calcScreen.innerHTML = '-';
```

Comment:
It's better to put `calcScreen.innerHTML = operation`. This way there's less of a chance for bugs.

The same for all operations below.